azerothcore/azerothcore-wotlk
# fix(Scripts/MoltenCore): Fixed some Geddon encounter issues

This PR fixes issues where Molten Core boss Geddon could make self interrupts and fail to cast Armageddon spell.
This PR also improves minor visual for Armageddon - once aura is applied, geddon will enter passive mode, will stop focusing on victim.

## Changes Proposed:
-  Fixed Geddon self interrupts
-  Improved Armageddon visual

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested by doing low damage by using damage command (.damage 10000) till he reached 2% hp, then observed his casts.
- Also i tested him normal casts without using damage command


## How to Test the Changes:


1. Try to engage Geddon
2. Observe his casts before 2%
3. You also need to observe his casts when Geddon is damaged to 2%, especially you need to observe Armageddon casts

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
